setup/ReShade Setup.csproj

```diff
@@ -37,6 +39,12 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.WindowsAPICodePack, Version=1.1.2.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
I completely understand, and I spent the better part of a day trying to find a native solution. Unfortunately, WPF does not provide a folder selection dialog, and the FolderBrowserDialog from win forms doesn't support multi-select. I could build a simple custom listview dialog, but that's a lot more work for a bad end result.

Everything I read online said to either roll your own, or to use that library. Given how much better the version provided by the dependency is, I opted for that. If you are strongly against adding dependencies, I'll make a simple custom one, but it will be a bare minimum implementation.